tests/io/test_file.py

```diff
@@ -22,6 +22,11 @@ def test_positionfixes_from_to_csv(self):
         pfs.as_positionfixes.to_csv(tmp_file, sep=';', columns=columns)
         assert filecmp.cmp(orig_file, tmp_file, shallow=False)
         os.remove(tmp_file)
+
+        crs = "EPSG:2056"
+        assert pfs.crs is None
+        pfs = ti.read_positionfixes_csv(orig_file, sep=';', index_col="id", crs=crs)
+        assert pfs.crs == crs
         
```

Comment:
Move code block into its own test.

For tests it is best if every test only tests a single specific functionality (see [atomic](https://defragdev.com/blog/2012/11/02/the-fundamentals-of-automated-testing-atomic.html) if you are interested). Our tests don't always adhere to this but we try our best ;-) 
This being said it would be great if you could paste the code into a new test. For this you probably have to copy parts of the test into the new one.
You could also replace the first assert `assert pfs.crs is None` with something that checks if there is a warning. (e.g., using the `with pytest.warns(UserWarning)` environment but it is up to you how exactly you create the test.